oximeter/db/src/db-replicated-init.sql

```diff
@@ -0,0 +1,190 @@
+CREATE DATABASE IF NOT EXISTS oximeter ON CLUSTER oximeter_cluster;
+--
+CREATE TABLE IF NOT EXISTS oximeter.measurements_bool ON CLUSTER oximeter_cluster
+(
+    timeseries_name String,
+    timeseries_key UInt64,
+    timestamp DateTime64(9, 'UTC'),
+    datum UInt8
+)
+
+ENGINE = ReplicatedMergeTree()
```

Comment:
Here are a few links I found from my search history, which I read through when initially making the DB organization. Not sure if they're useful, just wanted to make sure they're on your radar for this change:

- https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replication#converting-from-mergetree-to-replicatedmergetree
- https://clickhouse.com/docs/en/engines/table-engines/special/distributed
- https://disk.yandex.com/i/iLA5ssAv3NdYGy
- https://github.com/ClickHouse/ClickHouse/issues/2161

Reply:
Oh nice, thanks!! I'd seen some of these but not all :bowing_woman: